src/index.js

```diff
@@ -135,11 +131,12 @@ class Gofor {
     /**
      * Modify the default options
      * @param {Object} opts The new options to set.
+     * @return {self}
      */
     config(opts = null) {
-        this[defaultsKey] = this.mergeOptions(opts);
+        defineDefaults.call(this, opts);
 
-        return this[defaultsKey];
+        return this;
```

Comment:
While we're on it, this function doesn't feel like it should have a return value at all, since it's being used to define properties on the instance.
WDYT?

Reply:
Yeah, I'm not too attached to the fluent behaviour on this one